core/domain/user_jobs_one_off_test.py

```diff
@@ -434,7 +434,7 @@ def test_feedback_thread_subscription(self):
                 subscription_services, 'subscribe_to_exploration', self._null_fn
             ):
             with self.swap(
-                feconf, 'ENABLE_GENERALIZED_FEEDBACK_THREADS', False):
+                constants, 'ENABLE_GENERALIZED_FEEDBACK_THREADS', False):
```

Comment:
Why are these not in an "old_*_test.py" file?

What about the corresponding "new" tests?

Reply:
This one off job was mapping over FeedbackThreadModel (the old model). That's why I forced the constant to false. I am not sure, but I feel this job needs work on the new framework too? If so, I'll add tests (and change the job appropriately) for the new framework. But as of now, as the model under consideration was the old one, I had set the flag to false

Reply:
This job runs in a cron that is triggered weekly (cron.yaml). If you don't fix this properly for the new framework then that cron job will fail in production.

So this needs the same thorough treatment as the other cases. The URL handler called by the cron should redirect to the old job if the flag hasn't been flipped, and the new job if the flag has been flipped. There should be backend tests for each.

Reply:
Fixed the job, and also added tests for flag true and false cases.